tensorflow/core/kernels/cwise_op_gpu_equal_to.cu.cc

```diff
@@ -19,8 +19,8 @@ limitations under the License.
 
 namespace tensorflow {
 namespace functor {
-DEFINE_BINARY9(equal_to, float, Eigen::half, double, uint8, int8, int16, int64,
-               complex64, bool);
+DEFINE_BINARY10(equal_to, float, Eigen::half, double, uint8, int8, int16, int64,
+               complex64, bool, complex128);
```

Comment:
It a small thing, but perhaps put complex64 and complex128 together?  -> complex64, complex128, bool